src/third-party/PhishingController.ts

```diff
@@ -129,9 +128,7 @@ export class PhishingController extends BaseController<
   }
 
   /**
-   * Updates lists of approved and unapproved website origins
-   *
-   * @returns Promise resolving when this operation completes
+   * Updates lists of approved and unapproved website origins.
```

Comment:
Is the promise implied by `async`, is that why a `@returns` is not needed?

Reply:
Yes, exactly. I've tried to avoid comments that are re-explaining what the types already tell us, so anything that just says "this returns a string" or "this returns a promise" has been removed.

There are cases where docs are still relevant in cases like this, so it's not disallowed outright. For example you might want a `@returns` to explain that a function returns a Promise that resolves before the function completes, or a Promise that doesn't resolve until some long-running task completes, that kinda thing. And mentioning the type in the comment is still useful sometimes when the TypeScript type doesn't fully explain it (e.g. we might describe a parameter as a "hexidecimal string" when the type is just "string").

But if it doesn't convey anything useful beyond the type, I think we're better off removing it. It's one less line of documentation to keep up-to-date.